xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -143,15 +140,7 @@ The return value can be as follows:
         <summary>Attempts to decompress data that was compressed with the Brotli algorithm.</summary>
         <returns>
           <see langword="true" /> on success; <see langword="false" /> otherwise.</returns>
-        <remarks>
```

Comment:
I'm curious to see remarks like this: They were fully transformed to xml, but the newlines are probably not going to be respected in the preview. Which means I will have to modify DocsPortingTool so that it wraps every line with `<para></para>` (if the tool detects at least two paragraphs).

Reply:
Yep, we will need to add the `<para>` tags: https://review.docs.microsoft.com/en-us/dotnet/api/system.io.compression.brotlidecoder.decompress?view=net-5.0&branch=pr-en-us-5410#remarks

Reply:
*sigh* ... This will make our triple slash comments less readable. If that's the only choice we have, then I'll work on fixing the tool to address this.